apps/web/src/config/constants/ifo.ts

```diff
@@ -629,6 +629,28 @@ const ifos: Ifo[] = [
     tokenOfferingPrice: null,
     version: 1,
   },
+  {
+    id: 'test',
+    address: '0xF2F1E57D5EC65C206315803f83CCDce713E367c2',
```

Comment:
Before release, remember add this above `id: 'arena'` IFO

Reply:
Sure will replace the config